_i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md

```diff
@@ -0,0 +1,109 @@
+---
+title: How to learn and cope with negative thoughts  
+image: "https://guides.hexlet.io/ru/assets/images/learning/vs-negative-thoughts.png"
+author: Rakhim Davletkaliyev
+---
+
+Fitting active and regular exercise into your daily schedule may seem difficult, at first your body will resist: aching muscle pain, fatigue, unwillingness to continue. If you go for a run for the first time in decades, the body will clearly tell you: “you don't need to do this”. This is a natural reaction. An evolutionary defense tool against overloads and hazards.
+
+Any physical exercises or sports classes teach people to cope with this reaction and create their workout routine listening to the body needs.
+
+If you start active and regular mental exercises (for example, learning programming), then at first your brain will be very resistant. The brain has much more resistance tools than the muscles: it can plant thoughts that you're incompetent, that "this is not mine, and it's better for me to do something else," or "I'm not mathematically-minded" and so on.
+
+This article is about the mind’s resistance when faced with problems and overcoming these problems. It consists of four steps:
+
+1. **Realization.** Notice, realize, become a scientist.
+2. **Mindset.** Understand what success depends on.
+3. **Being open.** Exclude one of the main conditions of mental block.
+4. **Progress**. Accept the realities of the learning process.
+ 
+## Step 1: Realization
+
+First of all, it’s necessary to notice and realize the emerging resistance of the mind. Treat your body and mind as a scientific experiment: they give you, the observer, some signals, then they react to actions and environment.
+
+image: "https://guides.hexlet.io/assets/images/learning/learning_1.png"
+
+Try to notice or even write down the conditions under which such thoughts typically occur. A lot of factors affect the mental state, including physical condition, work environment, sleep, nutrition, stress, goals, relationships, etc.
+
+Keeping a journal can be a good idea. Write everything about your everyday life: how much you sleep, what do you do, what do you eat, how much do you read, how do you relax and how are your trainings going.
```

Comment:
how much you sleep, what do you do, what do you eat, how much do you read, how do you relax and how are your trainings going =>
how much you sleep, what you do, what you eat, how much you read, how you relax, and how is your training going.